chapters/cmdbuffers.txt

```diff
@@ -1362,7 +1362,7 @@ command buffer becomes <<commandbuffers-lifecycle, invalid>>.
   * [[VUID-vkCmdExecuteCommands-pInheritanceInfo-00098]]
     If fname:vkCmdExecuteCommands is being called within a render pass
     instance, the render passes specified in the
-    pname::pBeginInfo::pname:pInheritanceInfo::pname:renderPass members of
+    pname:pBeginInfo\->pname:pInheritanceInfo\->pname:renderPass members of
```

Comment:
Well, those are pointers.
Seemed to be rendered correctly. I need to retest #576 and https://github.com/asciidoctor/asciidoctor-pdf/issues/870 ...

Reply:
reverted tp `::`. I will to these at bulk, when I rebase #576.

Reply:
Frankly, I'd rather you write a script to do the bulk-editing. When there are PRs that simultaneously update many unrelated parts of the spec, they're difficult to integrate into our regular update process because accepting them is very likely to conflict with our internal MRs that are fixing specific parts of it.

Reply:
Its not gonna be so bad. Though I can, of course, give you my throwaway script too...